src/libsyncengine/jobs/network/abstracttokennetworkjob.cpp

```diff
@@ -293,6 +297,9 @@ std::string AbstractTokenNetworkJob::getUrl() {
         case ApiDriveByUser:
             apiUrl = KDRIVE_API_V2_URL;
             break;
+        case ApiDesktop: // TODO: Use KDRIVE_API_V2_URL when Desktop API is in production
+            apiUrl = DESKTOP_API_V2_URL;
```

Comment:
Will be remove as soon as the DESKTOP_API_V2_URL is in prod (in KDRIVE_API_V2_URL).